frontend/test/metabase/scenarios/question/trendline.cy.spec.js

```diff
@@ -0,0 +1,63 @@
+import {
+  restore,
+  signInAsNormalUser,
+  openOrdersTable,
+  sidebar,
+} from "__support__/cypress";
+
+describe("scenarios > question > trendline", () => {
+  before(restore);
+  beforeEach(signInAsNormalUser);
+
+  it.skip("displays trendline when there are multiple numeric outputs (for simple question) (Issue #12781)", () => {
+    // Create question: orders summarized with "Average of Subtotal" and "Sum of Total" by CreatedAt:Year
+    openOrdersTable();
+    cy.get(".Icon-notebook").click();
+    cy.findByText("Summarize").click();
+    cy.findByText("Average of ...").click();
+    cy.findByText("Subtotal").click();
+
+    cy.get(".Icon-add")
+      .last()
+      .click();
+    cy.findByText("Sum of ...").click();
+    cy.findByText("Total").click();
+
+    cy.findByText("Pick a column to group by").click();
+    cy.findByText("Created At").click();
+    cy.findByText("Created At: Month").click();
+    cy.findByText("by month").click();
+    cy.findByText("Year").click();
+
+    cy.findByText("Visualize").click();
+
+    // Check graph is there
+    cy.wait(1000).get("rect");
```

Comment:
Wouldn't `cy.get('rect')` retry for a few seconds? Why was the `wait(1000)` needed?

Reply:
The `wait(1000)` is because the test was timing out looking for `"rect"` I'll change it to look for another page element that loads faster than `"rect"` to improve the test.

Reply:
👍 Either way works for me! I was surprised that it's so slow, but I guess that is one of the slowest operations in the app.